### ISSUE TITLE
Property "inProcess" is no longer available while updating to 4.0.0

### ISSUE THREAD

**Issue by VolkerHartmann** [asciidoctor/asciidoctor-gradle-plugin#698]
Thank you very much for providing this great plugin.

However, when updating to the latest version (4.0.0) a problem has now arisen.
An error was thrown when building:
'Could not set unknown property 'inProcess' for task ':asciidoctor' of type org.asciidoctor.gradle.jvm.AsciidoctorTask.'

The new feature (executionMode) is documented in the current documentation. However, this is not yet online (https://asciidoctor.github.io/asciidoctor-gradle-plugin/).

Is there a migration guide? It would be helpful if it would at least appear in the release notes.

**Comment by ysb33r** [comment]
@abelsromero Do you know how the documentation is promoted to the Asciidoctor site?

**Comment by mojavelinux** [comment]
Not yet. In fact, the migration was never identified as completed. If it is, we can proceed. But all xrefs do need to be valid or else it will stop the build.

**Comment by abelsromero** [comment]
Yes, not in the site, for now just publish in gh pages as already done.
The issue back in the day was the docs are not "per branch" as the other repos, but docs for all versions are in a single one https://github.com/asciidoctor/asciidoctor-gradle-plugin/tree/gh-pages. I'd like to migrate to the other model if no one objects https://github.com/asciidoctor/asciidoctor-gradle-plugin/pull/601#issuecomment-1878369157. Some docs will be duplicated but the reduced mental load when keeping an eye on multiple repos is something I'd appreciate.

**Comment by ysb33r** [comment]
Closing this as in the upcoming 5.0.0-alpha.1 release we have a new model. 

I'll speak to @abelsromero to ensure the documentation will get out on time.
